libretroshare/src/rs_upnp/UPnPBase.cpp

```diff
@@ -369,6 +369,8 @@ m_direction           (upnpLib.Element_GetChildValueByTag(argument, "direction")
 m_retval              (upnpLib.Element_GetFirstChildByTag(argument, "retval")),
 m_relatedStateVariable(upnpLib.Element_GetChildValueByTag(argument, "relatedStateVariable"))
 {
+	/* remove unused private field warnings */
+	(void) m_UPnPControlPoint;

```

Comment:
I would really prefer that you keep the variables into /*  */ in the function prototype. The effect on compilation will be the same but it spares the use of a statement that does nothing. Maybe it's a matter of taste but I find this very ugly ;-)

Reply:
The same happens in many places in this PR. Can you update them all?

Reply:
These m_UPnPControlPoint is not used in our code but seems to be in UPnP one.
If not initialized, UPnP doesn't work.

Reply:
But you're not initializing here. You're just tricking the compiler in not giving a warning. This statement should just be removed.

Reply:
This commit is here to remove these warnings.
Do you have a better way to do it?